crm_helpdesk/data/crm_helpdesk_data.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<openerp>
+    <data noupdate="1">
+    <!--
+    Case Sections
+    -->
+    </data>
```

Comment:
I see... a pity to remove it, the nostalgia is very powerful :laughing: I would do it though!